<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_CadToolBarSelect</class>
 <widget class="QWidget" name="QG_CadToolBarSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>56</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>56</width>
    <height>336</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select</string>
  </property>
  <widget class="QToolButton" name="bAll">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select all</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectall.png</normaloff>:/ui/icons/selectall.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Back to main menu</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/back.png</normaloff>:/ui/icons/back.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bInters">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>104</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select intersected entities</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectinters.png</normaloff>:/ui/icons/selectinters.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bUnInters">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>104</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Deselect intersected entities</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/deselectinters.png</normaloff>:/ui/icons/deselectinters.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bUnAll">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Deselect all</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectnothing.png</normaloff>:/ui/icons/selectnothing.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bInvert">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>132</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Invert Selection</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectinvert.png</normaloff>:/ui/icons/selectinvert.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bLayer">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>132</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select layer</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectlayer.png</normaloff>:/ui/icons/selectlayer.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bContour">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>48</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>(De-)Select contour</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectcontour.png</normaloff>:/ui/icons/selectcontour.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bSingle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>48</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>(De-)Select entity</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectsingle.png</normaloff>:/ui/icons/selectsingle.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bUnWindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>76</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Deselect Window</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/deselectwindow.png</normaloff>:/ui/icons/deselectwindow.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bWindow">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>76</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select Window</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/selectwindow.png</normaloff>:/ui/icons/selectwindow.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bDoit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Continue action</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="png/icons.qrc">
     <normaloff>:/ui/icons/forward.png</normaloff>:/ui/icons/forward.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">rs_actioninterface.h</include>
  <include location="local">qg_actionhandler.h</include>
 </includes>
 <resources>
  <include location="png/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bSingle</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectSingle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bDoit</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>runNextAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBack</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAll</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bWindow</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bUnAll</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>deselectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bUnWindow</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>deselectWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bContour</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectContour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bUnInters</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>deselectIntersected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bInters</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectIntersected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bInvert</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectInvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bLayer</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarSelect</receiver>
   <slot>selectLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
